### DISCUSSION TITLE
[Feature suggestion] Fighter, Stage & more Expansion

### DISCUSSION THREAD

**Discussion by Bibidimah21** [Raytwo/ARCropolis#535] [Ideas]
Arcropolis, Adding new fighters, articles, items, etc. will possible coming to Arcropolis update like Paid DLC. Adding fighters used only Real CSS, adding stages used only Real SSS. Adding Real fighters mods, Real stage mods, no need to replace like Smash Brawl's BrawlEX. Diffrent Name folder inside the files of Adiditonal fighter/Stage.

**Comment by Bibidimah21** [comment]
Add Fan, Red Shell, Flipper & more items.

**Comment by Raytwo** [comment]
Weren’t you banned from ever posting here.
